src/bin/common/solving_loop.ml

```diff
@@ -485,7 +492,11 @@ let main () =
       pp_reason_unknown st
     | ":version" ->
       print_std Fmt.string Version._version
-    | ":all-statistics"
+    | ":all-statistics" ->
+      if Options.get_profiling () then
+        Printer.print_std "%t" Profiling.print_statistics
+      else
+        warning "Profiling disactivated (try --profiling)"
```

Comment:
In addition to the warning, there needs to be a reply to the standard output (e.g. `()` or maybe `unsupported`) that can be processed by automated tools.

Reply:
I updated the print: when profiling is off, no field is registered and `( )` is printed.